src/pages/library/Datasets.test.js

```diff
@@ -1,16 +1,18 @@
 import { render } from '@testing-library/react';
 import { h } from 'react-hyperscript-helpers';
-import { getEnabledBrand } from 'src/libs/brand-utils';
 import { brands } from 'src/libs/brands';
 import * as Nav from 'src/libs/nav';
 import { dataCatalogStore } from 'src/libs/state';
 import { prepareDatasetsForDisplay } from 'src/pages/library/dataBrowser-utils';
 import { Datasets } from 'src/pages/library/Datasets';
 
-jest.mock('src/libs/brand-utils', () => ({
-  ...jest.requireActual('src/libs/brand-utils'),
-  getEnabledBrand: jest.fn(),
-}));
+jest.mock('src/libs/brand-utils', () => {
+  const { brands } = jest.requireActual('src/libs/brands');
+  return {
+    ...jest.requireActual('src/libs/brand-utils'),
+    getEnabledBrand: jest.fn().mockReturnValue(brands.terra),
```

Comment:
libs/colors now requires `getEnabledBrand` to return a value when libs/colors is loaded. The current mock in this test causes `getEnabledBrand` to return undefined then.